e2e/logic/api/responses/getGraphsResponse.ts

```
@@ -0,0 +1,3 @@
+interface GetGraphsResponse {
```

ESLint

[error] 'GetGraphsResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)